proposals/nullable-reference-types-specification.md

```diff
@@ -129,24 +142,26 @@ The effect of the directives is as follows:
 
 # Nullability of types
 
-A `type` is deemed to occur in a given annotation context when the last token of the type is within that context.
+A given type can have one of four nullabilities: *Oblivious*, *nonnullable*, *nullable* and *unknown*. 
+
+*Nonnullable* and *unknown* types may cause warnings if a potential `null` value is assigned to them. *Oblivious* and *nullable* types, however, are "*null-assignable*" and can have `null` values assigned to them without warnings. 
+
+*Oblivious* and *nonnullable* types can be dereferenced or assigned without warnings. Values of *nullable* and *unknown* types, however, are "*null-yielding*" and may cause warnings when dereferenced or assigned without proper null checking. 
```

Comment:
This isn't right.  There should not be a warning assigning to a nullable variable.  It the null *nonnullable* variable that may yield a warning if the value being assigned might be null.